server/server.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, request, jsonify
+from controller.app_controller import AppController
+from model.repository.users_repository import UsersRepository
+
+app = Flask(__name__)
+
+
+@app.route('/api/v1/create_user', methods=['POST'])
+def create_user():
+    if request.method == 'POST':
+        # date = request.get_json()
+        user_id = request.json['user_id']
```

Comment:
You can put the requests.json value in another variable. It looks cleaner and you don't have to rewrite request.json over and over again